src/main/java/org/dita/dost/platform/Integrator.java

```diff
@@ -207,7 +208,8 @@ public void execute() throws Exception {
         for (final String tmpl : properties.getProperty(CONF_TEMPLATES, "").split(PARAM_VALUE_SEPARATOR)) {
             final String t = tmpl.trim();
             if (t.length() != 0) {
-                templateSet.add(t);
+                logger.warn(MessageUtils.getMessage("DOTJ080W", "templates", "template").toString());
```

Comment:
Is it OK to deprecate a common syntax in a hotfix, or are we supposed to leave that to the release boundary? (Seems a bit off with how I think about our rules / SemVer but I'm not certain)